package.json

```diff
@@ -12,31 +12,31 @@
     "releaseGithub": "conventional-github-releaser -p angular"
   },
   "devDependencies": {
-    "autoprefixer-stylus": "^0.13.0",
+    "autoprefixer-stylus": "^0.14.0",
     "babel-core": "^6.18.2",
     "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.2.7",
+    "babel-loader": "^7.1.0",
     "babel-plugin-transform-builtin-extend": "^1.1.0",
     "babel-plugin-transform-class-properties": "^6.19.0",
     "babel-plugin-transform-es2015-modules-amd": "^6.18.0",
     "babel-polyfill": "^6.16.0",
     "babel-preset-es2015": "^6.18.0",
     "browser-sync": "^2.18.1",
     "check-node-version": "^2.0.1",
-    "console-polyfill": "^0.2.3",
+    "console-polyfill": "^0.3.0",
     "conventional-github-releaser": "^1.1.3",
     "csso-loader": "^0.3.0",
-    "del": "^2.2.2",
+    "del": "^3.0.0",
     "document-register-element": "^1.3.0",
     "draggabilly": "^2.1.1",
-    "file-loader": "^0.10.1",
+    "file-loader": "^0.11.2",
     "glob": "^7.1.1",
     "gulp": "^3.9.1",
-    "gulp-changed": "^2.0.0",
+    "gulp-changed": "^3.1.0",
     "gulp-changed-in-place": "^2.0.3",
     "gulp-if": "^2.0.2",
     "gulp-notify": "^3.0.0",
-    "gulp-phpcs": "^1.3.0",
+    "gulp-phpcs": "^2.0.0",
```

Comment:
Nice, looks like we can get rid of this now: https://github.com/flyntwp/flynt-starter-theme/pull/192